src/test/rbd_mirror/test_PoolWatcher.cc

```diff
@@ -68,11 +69,12 @@ TestPoolWatcher() : m_lock("TestPoolWatcherLock"),
 
     m_pool_watcher.reset(new PoolWatcher(ioctx, 30, m_lock, m_cond));
     if (enable_mirroring) {
-      ASSERT_EQ(0, librbd::mirror_mode_set(ioctx, RBD_MIRROR_MODE_POOL));
+      ASSERT_EQ(0, librbd::api::Mirror<>::mode_set(ioctx,
+                                                   RBD_MIRROR_MODE_POOL));
       std::string uuid;
-      ASSERT_EQ(0, librbd::mirror_peer_add(ioctx, &uuid,
-					   peer.cluster_name,
-					   peer.client_name));
+      ASSERT_EQ(0, librbd::api::Mirror<>::peer_add(ioctx, &uuid,
+					           peer.cluster_name,
+					           peer.client_name));
```

Comment:
Looks correct in an editor -- just github not liking mixing tabs and spaces throughout the codebase